docs/_config.yml

```diff
@@ -0,0 +1,7 @@
+# site settings
+title: "☴ Mill"
```

Comment:
I mean, if it's just style, cool ... just not sure if that's a Jeckyll or githubpages thing.